ansible/plugins/inventory/adoptopenjdk_yaml.py

```diff
@@ -41,7 +41,7 @@
   'arch': ('armv7', 'armv8', 'ppc64le', 'ppc64', 'x64', 's390x', 'arm64', 'sparcv9'),
 
   # valid roles - add as necessary
-  'type': ('build', 'test', 'infrastructure', 'perf', 'docker'),
```

Comment:
Hmmm do we want to remove this while there's a possibility of getting more later? Gut feel would be leave it in for now so that it's easier for someone to add it in again later

Reply:
the inventory parser fails if there are no hosts in the specified group so I had to remove it. it will be very easy to add it back